float32 l_motor_val
float32 r_motor_val
bool relay_state
